components/curriculum/vars.css

```diff
@@ -162,32 +162,38 @@
-  --curriculum-landing-started-advanced: url("./assets/curriculum-landing-started-advanced.svg#light");
-  --curriculum-landing-started-employment: url("./assets/curriculum-landing-started-employment.svg#light");
-  --curriculum-landing-started-educator: url("./assets/curriculum-landing-started-educator.svg#light");
-  --curriculum-module-mdn-resource: url("./assets/curriculum-mdn-resource.svg#light");
-  --curriculum-bullet: url("./assets/curriculum-bullet.svg#light");
-  --curriculum-about-covered: url("./assets/curriculum-about-covered.svg#light");
-  --curriculum-about-detail: url("./assets/curriculum-about-detail.svg#light");
-  --curriculum-about-educators: url("./assets/curriculum-about-educators.svg#light");
-  --curriculum-about-not: url("./assets/curriculum-about-not.svg#light");
-  --curriculum-about-students: url("./assets/curriculum-about-students.svg#light");
-  --curriculum-bg-image-partner: url("./assets/curriculum-partner-bg.svg#light");
-  --curriculum-scrim-bg: url("./assets/curriculum-scrim-bg.svg#light");
-  --curriculum-landing-arrow: url("./assets/curriculum-landing-arrow.svg#light");
-
   --text-primary: var(--color-text-primary);
   --text-secondary: var(--color-text-secondary);
 }
 
+:root[data-theme="light"] {
+  .curriculum-content-container {
+    --curriculum-landing-about-beginner: url("./assets/curriculum-landing-about-beginner.svg#light");
+    --curriculum-landing-about-pace: url("./assets/curriculum-landing-about-pace.svg#light");
+    --curriculum-landing-about-free: url("./assets/curriculum-landing-about-free.svg#light");
+    --curriculum-landing-about-bullet: url("./assets/curriculum-landing-about-bullet.svg#light");
+    --curriculum-landing-started-beginner: url("./assets/curriculum-landing-started-beginner.svg#light");
+    --curriculum-landing-started-advanced: url("./assets/curriculum-landing-started-advanced.svg#light");
+    --curriculum-landing-started-employment: url("./assets/curriculum-landing-started-employment.svg#light");
+    --curriculum-landing-started-educator: url("./assets/curriculum-landing-started-educator.svg#light");
+    --curriculum-module-mdn-resource: url("./assets/curriculum-mdn-resource.svg#light");
+    --curriculum-bullet: url("./assets/curriculum-bullet.svg#light");
+    --curriculum-about-covered: url("./assets/curriculum-about-covered.svg#light");
+    --curriculum-about-detail: url("./assets/curriculum-about-detail.svg#light");
+    --curriculum-about-educators: url("./assets/curriculum-about-educators.svg#light");
+    --curriculum-about-not: url("./assets/curriculum-about-not.svg#light");
+    --curriculum-about-students: url("./assets/curriculum-about-students.svg#light");
+    --curriculum-bg-image-partner: url("./assets/curriculum-partner-bg.svg#light");
+    --curriculum-scrim-bg: url("./assets/curriculum-scrim-bg.svg#light");
+    --curriculum-landing-arrow: url("./assets/curriculum-landing-arrow.svg#light");
+    --test-theme: #ff0000;
```

Comment:
This variable (and the ones below) don't seem to be used anywhere, remove?

Reply:
These are used on the landing page.